draft-vvv-webtransport-overview.md

```diff
@@ -296,46 +302,47 @@ the size of the congestion window or the current pacing rate.
 ## Bandwidth Prediction
 
 Using congestion control state and transport metrics, the client can predict the
-rate at which it can send data.  That is essential for a lot of WebTransport use
+rate at which it can send data.  That is essential for many WebTransport use
 cases; for instance, real time media applications adapt the video bitrate to be
-a fraction of throughput they expect to be available.  While not all transport
-protocols can provide low-level transport details, any protocol SHOULD provide a
-way to estimate the bandwidth available to the client.
+a fraction of the throughput they expect to be available.  While not all
+transport protocols can provide low-level transport details, all protocols
+SHOULD provide the client with an estimate of the available bandwidth.
 
 # Buffering and Prioritization
 
 TODO: expand this outline into a full summary.
 
 * Datagrams are intended for low-latency communications, so the buffers for them
   should be small, and prioritized over stream data.
-* In general, the transport should not use any aggregation algorithms, e.g.
-  Nagle's algorithm {{?RFC0896}}.
+* In general, the transport should not apply aggregation algorithms (e.g.,
+  Nagle's algorithm {{?RFC0896}}) to datagrams.
```

Comment:
I was actually mostly worried about stream data.

Reply:
Nagle has value, I don't think we should argue against Nagle without a more detailed explanation

Reply:
Nagle does have value, and I'm pretty sure it's negative for ~all latency-sensitive applications we're targeting with this, at least without explicit controls.

Reply:
Not all streams are latency-sensitive in a latency-sensitive application though